admin/create-dumps.sh

```diff
@@ -85,20 +85,23 @@ else
     exit
 fi
 
-TMPDIR=$(mktemp --tmpdir="$TEMP_DIR" -d -t "$SUB_DIR.XXXXXXXXXX")
+DUMP_TEMP_DIR="$TEMP_DIR/$SUB_DIR.$$"
```

Comment:
Let's add a comment for what `$$` means.

Reply:
I see it refers to the shell's `PID`. What are the chances of a name conflict? I do not know how cron launches shell but I assume it will launch a new one always so I guess the question is do `PID`s get reused and if yes what is the possibility of the conflict?

Reply:
The use of $$ is an old skool trick to avoid conflicts -- generally in this context we don't need to worry about the PIDs rolling over since they would have to make an entire roll-over pass while dumps are being generated. That isn't going to happen. 
Alternatively we could use use a UUID, but then the UUID program might not be found and it dumps break yet again.